scripts/jenkins/jobs-ibs/openstack-mkcloud.yaml

```diff
@@ -0,0 +1,489 @@
+
+     - choice: 
+         name: cinder_backend
+         choices:
+           - local
+           - raw
+           - rdb
+           - netapp
+     - bool: 
+         name: want_cindermultibackend
+         default: false
+         description: 'enables deployment of more than one cinder backend'
+
+     - string:
+         name: cinder_netapp_login
+         default: 
+         description: the password for the user "openstack" as configured on the netapp server
+
+     - string:
+         name: cinder_netapp_password
+         default: 
+         description: the password for the user "openstack" as configured on the netapp server
+
+     - label:
+         name: label
+         default: openstack-mkcloud
+         description: |
+          Default: openstack-mkcloud | Possible values:
+          openstack-mkcloud-SLE11 (SLE11 host) | 
+          openstack-mkcloud-SLE12 (SLE12 host) | openstack-mkcloud-SLE12-aarch64 (SLE12 aarch64 host)
+
+     - string:
+         name: want_node_aliases
+         default: 
+         description: | 
+           List of aliases to assign to nodes. 
+           Takes all provided aliases and assign them to available nodes successively.
+           Examples want_node_aliases='controller=1,ceph=2,compute=1'
+           assigns the aliases to 4 nodes as controller, ceph1, ceph2, compute
+            want_node_aliases='data=1,services=2,storage=2'
+            assigns the aliases to 5 nodes as data, service1, service2,
+            storage1, storage2
+
+     - string:
+         name: scenario
+         default: 
+         description: | 
+          Defines which scenario file should be used. Only works with cloud5 or newer.
+          Currently only the 'batch' step uses such a file.
+
+     - string:
+         name: want_docker
+         default: 
+         description: set to 1 to deploy with docker
+
+     - string:
+         name: want_test_updates
+         default: 
+         description: allow disabling of test updates
+ 
+     - string:
+         name: want_magnum
+         default: 
+         description: Deploy with Magnum
+
+     - string:
+         name: want_barbican
+         default: 
+         description: Deploy with Barbican 
+
+     - string:
+         name: vcpus
+         default: '2'
+         description: number of vcpus for the compute nodes
+
+     - string:
+         name: want_sahara
+         default: 
+         description: set to 1 to deploy sahara
+
+     - string:
+         name: want_rootfs
+         default: 
+         description: set to btrfs to deploy with btrfs
+
+     - string:
+         name: want_postgresql
+         default: 
+         description: set to 1 to deploy with postgresql
+
+     - string:
+         name: want_efi
+         default: 
+         description: Set to UEFI boot admin node
+        
+     - string:
+         name: want_murano
+         default: 
+         description: set to 1 to deploy murano
+
```

Comment:
Around here there is a missing `custom_settings` multi-line parameter in the original